cmd/internal/codegen/parse/parser.go

```diff
@@ -62,11 +65,53 @@ func NewAPIs(context *generator.Context, arguments *args.GeneratorArgs) *APIs {
 	b.parseControllers()
 	b.parseRBAC()
 	b.parseInformers()
+	b.checkRbac()
 	b.parseAPIs()
 	b.parseCRDs()
 	return b
 }
 
+func (b *APIs) checkRbac() {
+	for gvk := range b.Informers {
+		found := false
+		for _, rule := range b.Rules {
+			groupFound := false
+			for _, g := range rule.APIGroups {
+				parts := strings.Split(g, ".")
+				group := parts[len(parts)-1]
+				if g == "*" || group == gvk.Group {
+					groupFound = true
+					break
+				}
+				if (group == "core" || group == "") && (gvk.Group == "core" || gvk.Group == "") {
+					groupFound = true
+					break
+				}
+			}
+			if !groupFound {
```

Comment:
Quick q: Don't we have to search in all the rules before declaring that group is not found ?

Reply:
Good catch